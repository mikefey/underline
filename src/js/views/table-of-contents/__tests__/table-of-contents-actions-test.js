import test from 'ava';
import {
  updateTableOfContents,
  showTableOfContents,
  hideTableOfContents,
} from './../table-of-contents-actions';

const data = [
  { title: 'item 1', href: 'item-1.html' },
  { title: 'item 2', href: 'item-2.html' },
];

test('updateTableOfContents action', (t) => {
  t.snapshot(updateTableOfContents(data));
});

test('showTableOfContents action', (t) => {
  t.snapshot(showTableOfContents());
});

test('hideTableOfContents action', (t) => {
  t.snapshot(hideTableOfContents(data));
});
