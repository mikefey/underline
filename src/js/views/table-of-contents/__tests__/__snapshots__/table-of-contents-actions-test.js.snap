exports[`hideTableOfContents action 1`] = `"{"type":"HIDE_TABLE_OF_CONTENTS"}"`;

exports[`showTableOfContents action 1`] = `"{\"type\":\"SHOW_TABLE_OF_CONTENTS\"}"`;

exports[`updateTableOfContents action 1`] = `"{\"type\":\"UPDATE_TABLE_OF_CONTENTS\",\"data\":[{\"title\":\"item 1\",\"href\":\"item-1.html\"},{\"title\":\"item 2\",\"href\":\"item-2.html\"}]}"`;
